src/main/webapp/mapScript.js

```diff
@@ -107,8 +108,12 @@ function fetchPlaceInformation(place_id) {
 /** Makes place_id and location name of a place available. */
 function getLocationInfo() {
   locationInputElement = document.getElementById('location');
+  placeIdInputElement = document.getElementById('placeId');
   locationName = sessionStorage.getItem('locationName');
+  placeId = sessionStorage.getItem('placeId');
+  console.log(placeId);
```

Comment:
Was this for testing or is there a reason we are printing the place id to the console?

Reply:
it was just for testing. Thanks for pointing that out

Reply:
no prob, it doesn't matter too much I just wanted to make sure I wasn't missing something!